ecs-cli/modules/utils/compose/ecs_params_reader.go

```diff
@@ -104,6 +105,49 @@ type AwsVpcConfiguration struct {
 
 type AssignPublicIp string
 
+// ServiceDiscovery hold information related to ECS/Route53 Service Discovery
+type ServiceDiscovery struct {
+	ContainerName           string                  `yaml:"container_name"`
+	ContainerPort           *int64                  `yaml:"container_port"`
+	PrivateDNSNamespace     PrivateDNSNamespace     `yaml:"private_dns_namespace"`
+	PublicDNSNamespace      PublicDNSNamespace      `yaml:"public_dns_namespace"`
+	ServiceDiscoveryService ServiceDiscoveryService `yaml:"service_discovery_service"`
+}
+
+// PrivateDNSNamespace holds information related to Route53 private DNS namespaces
+type PrivateDNSNamespace struct {
+	VPC         string `yaml:"vpc"`
+	ID          string `yaml:"id"`
+	Name        string `yaml:"name"`
+	Description string `yaml:"description"`
+}
+
+// PublicDNSNamespace holds information related to Route53 public DNS namespaces
+type PublicDNSNamespace struct {
+	ID   string `yaml:"id"`
+	Name string `yaml:"name"`
+}
+
+// ServiceDiscoveryService holds information related to Route53 Service Discovery Services
+type ServiceDiscoveryService struct {
+	Name                    string                  `yaml:"name"`
+	Description             string                  `yaml:"description"`
+	DNSConfig               DNSConfig               `yaml:"dns_config"`
+	RoutingPolicy           string                  `yaml:"routing_policy"`
+	HealthCheckCustomConfig HealthCheckCustomConfig `yaml:"healthcheck_custom_config"`
+}
+
+// DNSConfig holds the dns configuration for Service Discovery Services
+type DNSConfig struct {
+	Type string `yaml:"type"`
+	TTL  *int64 `yaml:"ttl"`
+}
+
+// HealthCheckCustomConfig
+type HealthCheckCustomConfig struct {
+	FailureThreshold *int64 `yaml:"failure_threshold"`
+}
```

Comment:
Does `HealthCheckCustomConfig` contain other fields we plan on supporting in the future?